benchmarks/runBench.js

```diff
@@ -1,16 +1,46 @@
 'use strict';
 console.log('----------Starting tests-------');
-require('./add.js');
-console.log('----------');
-require('./mul.js');
-console.log('----------');
-require('./subtract.js');
-console.log('----------');
-require('./divide.js');
-console.log('----------');
-require('./abs.js');
-console.log('----------');
-require('./compare.js');
-console.log('----------');
-require('./remainder.js');
-console.log('----------');
+
+const suites = [
+    require('./constructor'),
```

Comment:
As we have a lot of benchmarks now, can you please move these files in a folder maybe named "methods" or something like that? We have other files like runBench utils or testNumbers and it may be hard to find them.